src/components/DisplayComponents/Display.js

```diff
@@ -1,5 +1,10 @@
 import React from "react";
 
 const Display = () => {
-  return <div>{/* Display any props data here */}</div>;
+  return 
+  
+  <div>{/* Display any props data here */}
+  
+  </div>;
```

Comment:
We would want this to take in a number or string as a prop, and then display that inside of the div, so it works as our calculator string.